syntax = "proto3";

package test;

service HelloWorld{
    rpc show_msg(HelloGrpcReq) returns (HelloGrpcReply) {}
}

message HelloGrpcReq {
    string name = 1;
    int32 age = 2;
}

message HelloGrpcReply {
    string result = 1;
}